native/src/components/CalendarChoice.tsx

```diff
@@ -0,0 +1,71 @@
+import { HeaderBackButton } from '@react-navigation/elements'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Modal, TouchableOpacity } from 'react-native'
+import { Calendar } from 'react-native-calendar-events'
+import styled from 'styled-components/native'
+
+const StyledView = styled.View`
+  flex: 1;
+  justify-content: center;
+  margin: 0 20px;
+  padding: 50px 0;
+`
+
+const TitleContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 5px;
+`
+
+const Title = styled.Text`
+  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
+  text-align: center;
+  flex: 1;
+  padding-right: 16px;
+`
+
+const CalendarButton = styled(TouchableOpacity)<{ color: string }>`
+  flex: 0.1;
+  background-color: ${props => props.color};
+  justify-content: center;
+  align-items: center;
+  margin: 5px 0;
+`
+
+const ButtonText = styled.Text`
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+`
+
+type CalendarChoiceProps = {
+  calendars: Calendar[]
+  chooseCalendar: (id: string) => void
+  overlayVisible: boolean
+  closeOverlay: () => void
+}
+
+const CalendarChoice = ({
+  calendars,
+  chooseCalendar,
+  overlayVisible,
+  closeOverlay,
+}: CalendarChoiceProps): ReactElement => {
+  const { t } = useTranslation('events')
+  return (
+    <Modal visible={overlayVisible} onRequestClose={closeOverlay} animationType='fade'>
+      <StyledView>
+        <TitleContainer>
+          <HeaderBackButton onPress={closeOverlay} label={t('back')} />
+          <Title>{t('chooseCalendar')}</Title>
+        </TitleContainer>
```

Comment:
I think we should use the same header style (same buttons, shadows, header text style...) as for our other headers, the following would work if you export the components from `TransparentHeader`:
```suggestion
      <BoxShadow>
        <Horizontal>
          <HorizontalLeft>
            <HeaderBackButton onPress={closeOverlay} labelVisible={false} tintColor={theme.colors.textColor} />
          </HorizontalLeft>
        </Horizontal>
      </BoxShadow>
      <StyledView>
        <Title>{t('chooseCalendar')}</Title>
```
However, since we use and define the same elements two times already (Header, TransparentHeader), it might also make sense to create a new component `HeaderBox`, which takes a `HeaderLeft` component and optional text and header buttons as props which can then be used in TransparentHeader, Header and here.